src/js/covidMap.js

```diff
@@ -0,0 +1,477 @@
+/* global fluid, L */
+
+"use strict";
+
+fluid.setLogging(true);
+
+fluid.registerNamespace("fluid.mapviz.covid");
+
+fluid.mapviz.covid.metresInMile = 1609.34;
+
+fluid.defaults("fluid.mapviz.covid.pagerBar", {
+    gradeNames: "fluid.pager.pagerBar",
+    components: {
+        pageList: {
+            type: "fluid.emptySubcomponent"
+        }
+    },
+    selectors: {
+        previous: ".fld-mapviz-page-left",
+        next: ".fld-mapviz-page-right"
+    }
+});
+
+fluid.defaults("fluid.mapviz.covid.pager", {
+    gradeNames: "fluid.pager",
+    dynamicComponents: {
+        pagerBar: {
+            type: "fluid.mapviz.covid.pagerBar"
+        },
+        summary: {
+            options: {
+                strings: {
+                    message: "Showing results %first-%last of %total"
+                }
+            }
+        }
+    },
+    modelRelay: {
+        // These definitions taken from Pager's summary subcomponent, should go into its definition
+        firstIndex: {
+            target: "firstIndex",
+            func: (pageIndex, pageSize) => pageIndex * pageSize + 1,
```

Comment:
This is cool. Great to learn such way to attach an actual function in the component declaration. Thanks to the power of ES6 and infusion.